src/send/utils.ts

```diff
@@ -135,7 +136,7 @@ export function* handleSendPaymentData(
 export function* handlePaymentDeeplink(deeplink: string) {
   try {
     const paymentData = uriDataFromUrl(deeplink)
-    yield call(handleSendPaymentData, paymentData)
+    yield call(handleSendPaymentData, paymentData, false)
```

Comment:
aren't payment deep links usually from QR codes that users scan?

Reply:
Yeah good catch, changed to isFromScan to true